experimental/cotel/otel.go

```diff
@@ -20,13 +20,9 @@ func AddSpanWithOpts(
 		return ctx
 	}
 
-	ctx, spanned := nc.AddSpan(ctx, name, opts...)
-
-	if len(kvs) > 0 {
-		spanned.ID = ""
-		spanned = spanned.AddValues(ctx, stringify.Normalize(kvs...))
-		spanned.ID = name
-	}
-
-	return node.EmbedInCtx(ctx, spanned)
+	return nc.AddSpan(
+		ctx,
+		name,
+		stringify.Normalize(kvs...),
```

Comment:
I think maybe you forgot to add `opts` to the parameter set?

```suggestion
	return nc.AddSpan(
		ctx,
		name,
		stringify.Normalize(kvs...),
		opts...,
```

Reply:
argle, thank you